### ISSUE TITLE
Suche nach Benutzernamen in Backend-Buchung-Liste funktioniert nicht

### ISSUE THREAD

**Issue by hanneskasimir** [wielebenwir/commonsbooking#909]

**Comment by hansmorb** [comment]
Meinst du die Freitextsuche rechts oben? Die sucht nur nach Buchungstiteln, andere Parameter kann die nicht. Einen Filter für Nutzende im Backend wäre vielleicht möglich, ich bin mir aber nicht sicher ob dann bei Installationen mit sehr vielen Nutzenden die Buchungsliste noch richtig lädt. Im Frontend gibt es dafür eine Lösung.

**Comment by hanneskasimir** [comment]
Ja, die meine ich. Schade, hatte gehofft, dass das einfach erweiterbar wäre im Backend.

Vielleicht muss ich nochmal kurz ausholen:
Ich war auf der Suche nach einer Buchungen einer bestimmten Nutzerin, d.h. eine vergangene Buchung. 
Im Frontend sind vergangene Buchungen aktuell nicht drin, spricht was dagegen, die dort mit reinzunehmen?

**Comment by hansmorb** [comment]
Es gibt die Möglichkeit im Frontend das Buchungsdatum auf ein Datum in der Vergangenheit zu setzen, dann sollten auch frühere Buchungen angezeigt werden.

**Comment by hanneskasimir** [comment]
ah, klar, ich Depp. Danke @hansmorb !
